### ISSUE TITLE
Unable to move after blob smoke hit.

### ISSUE THREAD

**Issue by Firecage** [tgstation/-tg-station#9283]
Well. I was the CMO and was dragging a cuffed dude to escape. It was then that the smoke from one of those killed blob spores hit me, the one which knocks you back. Afterwards I couldn't move at all, even though I could interact with everything. Thus didn't fade and lasted until the round ended.

Ausops checked me and said I had no reagents in me, and no active status effects.

**Comment by Iamgoofball** [comment]
Weird as fuck

Can someone try to reproduce this

**Comment by Alek2ander** [comment]
You were probably buckled in some weird way

**Comment by Firecage** [comment]
Alek2ander. Though resist did nothing. I could pick stuff up. Interact with everything. And people could pull me around. I was just...immovable.

**Comment by peoplearestrange** [comment]
I'm fairly certain this is when a "Morphine" blob is created (as I saw last night). The effects of an exploding blobling spore seem to knock people about. Also the effect of the morphine is INCREDIBLY OP, with most people in the round dying of OD whilst trying to attack the blob. Even if they were ranged the blobspores would attack them, explode, and give them morphine via the smoke. I also saw someone mention that the gas masks do nothing to prevent this either.
